ecommerce/PaymentHandler.php

```diff
@@ -80,14 +70,16 @@ public function handle_ecommerce_webhook( WP_REST_Request $request ) {
 		);
 		// phpcs:enable
 
-		$route          = $request->get_route() ?? null;
-		$payment_method = ! is_null( $route ) && strpos( $route, self::AUTHORIZENET ) ? self::AUTHORIZENET : Input::get( 'payment_method', 'paypal' );
+		$payment_method = $request->get_param( 'payment_method' ) ?? $request->get_param( 'payment_gateway' ) ?? '';
```

Comment:
After changing the above we can use just this: `$request->get_param( 'payment_method' )`